lib/seahorn/Smt/TypeChecker.cc

```diff
@@ -12,64 +12,72 @@ namespace {
 
 //==-- main implementation goes here --==//
 class TCVR {
-  ExprVector expTypeStack;
-
-  bool isConst (Expr exp) {
-   return bind::isBoolConst(exp) || bind::isIntConst(exp);
+  ExprMap exprMap;
```

Comment:
filed names should start with `m_`. A good name would be `m_cache`.
https://github.com/seahorn/seahorn/wiki/Developer-Corner